src/components/content/deploymentVariables/DeploymentVariables.tsx

```diff
@@ -0,0 +1,233 @@
+                                </div>
+                            </pre>
+                        }
+                        title={'Description'}
+                        trigger='hover'
+                    >
+                        <Button
+                            className={deploymentVariablesStyles.variablesDataHover}
+                            type={'link'}
+                        >{`description`}</Button>
+                    </Popover>
+                );
+            },
+        },
+        {
+            title: <div className={deploymentVariablesStyles.variablesColumns}>Default</div>,
+            dataIndex: 'value',
+            render: (text: string) => {
+                return <div className={deploymentVariablesStyles.variablesColumns}>{text}</div>;
+            },
+        },
+        {
+            title: <div className={deploymentVariablesStyles.variablesColumns}>Mandatory</div>,
+            dataIndex: 'mandatory',
+            render: (text: boolean | undefined | null) => {
+                if (text === true) {
+                    return <div className={deploymentVariablesStyles.variablesColumns}>{`true`}</div>;
+                } else if (text === false) {
+                    return <div className={deploymentVariablesStyles.variablesColumns}>{`false`}</div>;
+                } else {
+                    return null;
+                }
+            },
+        },
+        {
+            title: <div className={deploymentVariablesStyles.variablesColumns}>ValueSchema</div>,
+            dataIndex: 'valueSchema',
+            render: (_text: string, record: DeployVariable) => {
+                if (!record.valueSchema) {
+                    return '';
+                }
+
+                const formattedValueSchema = Object.entries(record.valueSchema)
+                    .map(([key, value]) => `${key}: ${value as string}`)
+                    .join('\n');
+
+                return (
+                    <Popover
+                        content={
+                            <pre>
+                                {' '}
+                                <div className={deploymentVariablesStyles.variablesContentHover}>
+                                    {formattedValueSchema}
+                                </div>
+                            </pre>
+                        }
+                        title={'Value Schema'}
+                        trigger='hover'
+                    >
+                        <Button
+                            className={deploymentVariablesStyles.variablesDataHover}
+                            type={'link'}
+                        >{`valueSchema`}</Button>
+                    </Popover>
+                );
+            },
+        },
+        {
+            title: <div className={deploymentVariablesStyles.variablesColumns}>SensitiveScope</div>,
+            dataIndex: 'sensitiveScope',
+            render: (text: 'none' | 'once' | 'always') => {
```

Comment:
we must avoid having this list here delcared once more here. Must be used directly from the generated code.

Reply:
![image](https://github.com/user-attachments/assets/621c49c5-7d40-4988-97e0-0a0ff731ec0f)

![image](https://github.com/user-attachments/assets/f0aa2e39-2a15-4af6-b2d3-0b4d39d45559)

fixed.